### PR TITLE
gnome-extra/gnome-commander: Version bump to 1.8.1

### PR DESCRIPTION
**Pull request by turboscholz**
Removed version 1.6.1 from portage.

### CODE REVIEW

**Comment by turboscholz** [comment]
The bug report for the version bump has been created: https://bugs.gentoo.org/654522

File: gnome-extra/gnome-commander/Manifest [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DIST gnome-commander-1.6.1.tar.xz 6550308 BLAKE2B d4353bce913a43e091f2d66b16c932ff8c3d863a0dd323f487a8ef4b9ffacbe4e62e70600d4fb8e5ca15759c3c07888a17662e328abbbb7013446d294986c11d SHA512 717e8bb560c6c7e6505cfee1d9ab12f7b3d5c2f8131b9eaf8b15e13aa71dad85574b14ae6f3a42dae42d8a6a291d33cd71db5a4155184bec4f66e3db8b2daab0
```
Comment (mgorny) [inline gnome-extra/gnome-commander/Manifest]: Please split this into two commits, one adding the new version, and the other removing old.

Comment (turboscholz) [reply gnome-extra/gnome-commander/Manifest]: Okay, will do that. Shall I open a bug report for the removal of version 1.6.1?

Comment (mgorny) [reply gnome-extra/gnome-commander/Manifest]: No. Bug reports are not necessary for the packages you're maintaining.

**Comment by mgorny** [comment]
You're still removing the old ebuild.

**Comment by turboscholz** [comment]
Oh, thanks. My fault... I fixed it.

File: gnome-extra/gnome-commander/gnome-commander-1.8.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GNOME2_LA_PUNT="yes"
+
+inherit eutils gnome2
```
Comment (mgorny) [inline gnome-extra/gnome-commander/gnome-commander-1.8.1.ebuild]: You don't seem to be using eutils here. I will fix it for you and merge.

**Pull request closed without merging**